admin/code/CMSProfileController.php

```diff
@@ -57,7 +57,7 @@ public function canView($member = null) {
 		
 		// Only check for generic CMS permissions
 		if(
-			!Permission::checkMember($member, "CMS_ACCESS_LeftAndMain")
```

Comment:
Note, this has been removed as a recent 3.1 fix has made `CMS_ACCESS_*` implicitly `CMS_ACCESS_LeftAndMain`